spec/unit/application/lookup_spec.rb

```diff
@@ -89,7 +89,6 @@ def run_lookup(lookup)
     end
   end
 
-
```

Comment:
seems this file could be left unchanged.

Reply:
sure, but it is no harm removing that extra empty line anyway :-)